### ISSUE TITLE
CustomListItem: No aria-label support

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents-react#1449]
**Describe the bug**
When using the following component within List, we cannot using aria-label to define a message for a11y use case.
So there will be no indication that if there are interactive elements within each list item or not. 
- StandardListItem
- CustomListItem
- GroupHeaderListItem

**Expected behavior**
aria-label support for the following component
- StandardListItem
- CustomListItem
- GroupHeaderListItem

**Additional context**
I am from SAP SuccessFactors

**Comment by MarcusNotheis** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected components are developed in their repository.
